src/test/java/org/gridsuite/modification/server/ModificationControllerTest.java

```diff
@@ -1208,7 +1069,7 @@ private List<TapChangerStepCreationInfos> getTapChangerSteps() {
         );
     }
 
-    private void testCreateTwoWindingsTransformerInNodeBreaker(TwoWindingsTransformerCreationInfos twoWindingsTransformerCreationInfos) throws Exception {
+    private void testCreateTwoWindingsTransformerInNodeBreaker(TwoWindingsTransformerCreationInfos twoWindingsTransformerCreationInfos, int actualSize) throws Exception {
```

Comment:
Move all tests `testCreateTwoWindingsTransformer*` in TwoWindingsTransformerCreationTest class